### PR TITLE
Implement Rakdos Firewheeler

### PR DESCRIPTION
**Pull request by jmharmon**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/r/RakdosFirewheeler.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        this.subtype.add(SubType.ROGUE);
+
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(3);
+
+        Effect effect = new DamageTargetEffect(2);
+        effect.setText("it deals 2 damage to target opponent and 2 damage to up to one target creature or planeswalker");
+        Ability ability = new EntersBattlefieldTriggeredAbility(effect, false);
+        ability.addTarget(new TargetOpponent());
+        ability.addTarget(new TargetCreatureOrPlaneswalker());
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/r/RakdosFirewheeler.java]: This needs to be "up to" one target creature or planeswalker, so you need to set the minimum to 0.

File: Mage.Sets/src/mage/cards/r/RakdosFirewheeler.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    public RakdosFirewheeler(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{B}{B}{R}{R}");
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.ROGUE);
+
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(3);
+
+        Effect effect = new DamageTargetEffect(2);
```
[the remark this answers is not in the document]
Comment (theelk801) [reply Mage.Sets/src/mage/cards/r/RakdosFirewheeler.java]: If you use `gen-card.pl` in the Utils folder you'll be able to do this automatically.

**Comment by theelk801** [comment]
You also need to add the card to the Ravnica Allegiance set file

**Comment by jmharmon** [comment]
Card update, I don't think the set file is updated with other changes.

**Pull request merged**